ext/native/gfx_es2/gpu_features.cpp

```diff
@@ -335,6 +335,7 @@ void CheckGLExtensions() {
 	gl_extensions.EXT_blend_func_extended = g_set_gl_extensions.count("GL_EXT_blend_func_extended") != 0;
 	gl_extensions.ARB_conservative_depth = g_set_gl_extensions.count("GL_ARB_conservative_depth") != 0;
 	gl_extensions.ARB_shader_image_load_store = (g_set_gl_extensions.count("GL_ARB_shader_image_load_store") != 0) || (g_set_gl_extensions.count("GL_EXT_shader_image_load_store") != 0);
+	gl_extensions.ARB_shading_language_420pack = (g_set_gl_extensions.count("GL_ARB_shading_language_420pack") != 0);
```

Comment:
We should also set this to true down lower, see here:
https://github.com/hrydgard/ppsspp/blob/27d216ea25cf56cec32ec075028316c9167c0d91/ext/native/gfx_es2/gpu_features.cpp#L516

Since AFAIK, this extension can be elided in the extensions string on 4.2+.

-[Unknown]

Reply:
I am not sure if it is intended. Since 4.2 it is the core functionality indeed, yet the extension support has nothing to do to it. For instance, SPIR-V cross has the following code, and it makes good sense to me:
```C++
can_use_binding = options.enable_420pack_extension || (options.version >= 420);
```
I am fine to change it if `gl_extensions` is just mislabeled and does not mean extension availability but rather OpenGL features. However, I would suggest to rename it to `gl_features` in this case.